en/docs/tutorials/integrating-services-tutorial.md

```diff
@@ -0,0 +1,812 @@
+    
+    2. Enter information for the variable as follows:
+
+        | **Field**         | **Value**                    |
+        |-------------------|------------------------------|
+        | **Type**          | **string**                   |     
+        | **Name**          | `inventoryItemId`            |
+        | **Expression**    | `queryParams.get("item")[0]` |
+    
+    3. Click **Save**.
+    
+7. To enable Choreo to read the value for the `quantity` parameter in the order request, add another variable statement as follows:
+
+    1. Click the **+** icon below the `inventoryItemId` variable statement you added, and then click **Variable**. 
+    
+    2. Enter information for the variable as follows:
+
+        | **Field**         | **Value**                                           |
+        |-------------------|-----------------------------------------------------|
+        | **Type**          | **int**                                             |     
+        | **Name**          | `quantity`                                          |
+        | **Expression**    | `check int:fromString(queryParams.get("count")[0])` |
+        
+    3. Click **Save**.
+     
+ 8. To connect to the Google sheet where inventory records are maintained, add an API call as follows: 
+            
+    1. Click the **+** below the `quantity` variable statement you added, and then click **API Calls**.
+    
+    2. Click **Google Sheets**
+    
+    3. Click **Connect to Google Sheets**, and then click **Proceed**. Click on the Google Account of your choice and click **Allow** too allow Choreo to access it as described in the page. Then click **Save**.
+    
+    4. Click the **+** icon below the Google Sheets connection you just added. Click **API Calls**, and then click **sheetsEndpoint** under **Choose existing connection**.
+    
+    5. In the **Operation** field, select **Get values from a column**. Then enter information as follows in the rest of the fields that appear:
+    
+        | **Field**                  | **Value**                                                                                                       |
+        |----------------------------|-----------------------------------------------------------------------------------------------------------------|
+        | **SpreadsheetId**x         | The ID of your google sheet.                                                                                    |  
+        | **Worksheet Name**         | The name of the sheet with the inventory records.                                                               |
+        | **Column**                 | The ID of the column in which you have added values for the `item` variable (i.e., `A` in the given example image). |
+        | **Response Variable Name** | `itemIDs`                                                                                                       |
```

Comment:
Once we complete step 5 we get error `variable assignment is required`. We have to add `(string|int|float)[] listResult = ` by clicking quick fix.

Reply:
Since this is an error, I reported two issues for this as discussed. The issues are:

- https://github.com/wso2-enterprise/choreo/issues/4863
- https://github.com/wso2-enterprise/choreo/issues/4864